lib/installation-proxy/index.js

```diff
@@ -5,6 +5,21 @@ class InstallationProxyService {
     this.plistService = plistService;
   }
 
+  /**
+   * Install the application on the relative path on the phone
+   * @param {string} path The path where the .app and .ipa is located at on the phone
+   * @param {number} timeout The timeout between messages received from the phone as status updates
+   */
+  async installApplication (path, timeout = 20000) {
```

Comment:
I'd rather set the default value to 60k or 120k. There are really large apps there